.env

```diff
@@ -0,0 +1,75 @@
+#  * .env.local          uncommitted file with local overrides
+#  * .env.$APP_ENV       committed environment-specific defaults
+#  * .env.$APP_ENV.local uncommitted environment-specific overrides
+#
+# Real environment variables win over .env files.
+#
+# DO NOT DEFINE PRODUCTION SECRETS IN THIS FILE NOR IN ANY OTHER COMMITTED FILES.
+#
+# Run "composer dump-env prod" to compile .env files for production use (requires symfony/flex >=1.2).
+# https://symfony.com/doc/current/best_practices/configuration.html#infrastructure-related-configuration
+
+###> symfony/framework-bundle ###
+APP_ENV=dev
+APP_SECRET=7fd781d3c8b25811ddd82260d88cc496
+#TRUSTED_PROXIES=127.0.0.1,127.0.0.2
+#TRUSTED_HOSTS='^localhost|example\.com$'
+###< symfony/framework-bundle ###
+
+###> doctrine/doctrine-bundle ###
+# Format described at https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
+# For an SQLite database, use: "sqlite:///%kernel.project_dir%/var/data.db"
+# Configure your db driver and server_version in config/packages/doctrine.yaml
+DATABASE_URL=postgres://user:password@127.0.0.1:3306/db_name
+###< doctrine/doctrine-bundle ###
+
+###> php-amqplib/rabbitmq-bundle ###
+RABBITMQ_URL=amqp://guest:guest@localhost:5672
+###< php-amqplib/rabbitmq-bundle ###
+
+###> symfony/swiftmailer-bundle ###
+# For Gmail as a transport, use: "gmail://username:password@localhost"
+# For a generic SMTP server, use: "smtp://localhost:25?encryption=&auth_mode="
+# Delivery is disabled by default via "null://localhost"
+MAILER_URL=null://localhost
+###< symfony/swiftmailer-bundle ###
+
+###> parameters ###
+HOSTNAME=host-env.tamucc.edu
+GOOGLE_MAPS_API_KEY=SECRET
+GOOGLE_ANALYTICS_ID=ID
+PELAGOS_READONLY_MESSAGE=MESSAGE
+MAINTENANCE_MODE_FILENAME="/var/maintenance.ini"
+ACCOUNT_PASSWORD_MAX_AGE=P180D
+ACCOUNT_PASSWORD_EXPIRATION_WARN=P7D,P1D
+LDAP_HOST=localhost
+LDAP_BIND_DN='cn=admin,dc=domain,dc=org'
+LDAP_BIND_PW='secretpassword'
+LDAP_PEOPLE_OU='ou=users,dc=domain,dc=org'
+ACCOUNT_LESS_STRICT_PASSWORD_RULES=FALSE
+MAILER_FROM_ADDR='email@domain.null'
+MAILER_FROM_NAME='SenderName'
+PELAGOS_NAME=pelagos
+ISSUE_TRACKING_BASE_URL='https://griidc.atlassian.net/issues'
+DATABASE_NAME=pelagos
+DATA_STORE_DIRECTORY=/path/to/store/directory
+DATA_DOWNLOAD_DIRECTORY=/path/to/download/directory
+DATA_STORE_OWNER=user
+DATA_STORE_GROUP=group
+DATA_DOWNLOAD_BROWSER_GROUP=browser_group
+WEB_SERVER_USER=apache
+ANONYMOUS_FTP_USER=user
+ANONYMOUS_FTP_PASS=pass
+SEARCH_TOOL_INDEX=search_instancename
+HOMEDIR_PREFIX=/path/to/home/directory/prefix
+MAINTENANCE_MODE=false
+DOWNLOAD_BASE_DIRECTORY=/path/to/http/download
+# The DOWNLOAD_BASE_URL parameter target also must be configured for direct file access on the Webserver.
+DOWNLOAD_BASE_URL=https://server-name/path/file-download
+POSIX_STARTING_UID=10000
+POSIX_GID=1001
+PELAGOS_BASE='/pelagos-symfony'
```

Comment:
Made add URL?

Reply:
just for future use?

Reply:
Sorry, that sounded confusing, the type-o didnt' help. I meant, add _URL the this parameter, because it's setting something that is called .....base_url.  Even tho that's not what it is, that's what it's called.